src/docdb/tree/DocDBDocumentTreeItem.ts

```diff
@@ -18,23 +19,24 @@ export class DocDBDocumentTreeItem implements IAzureTreeItem {
     public readonly contextValue: string = DocDBDocumentTreeItem.contextValue;
     public readonly commandId: string = 'cosmosDB.openDocument';
 
-    private readonly partitionKeyValue: string | undefined | Object;
-
+    private readonly _partitionKeyValue: string | undefined | Object;
+    private _label: string;
     private _document: RetrievedDocument;
     private _collection: DocDBCollectionTreeItem;
 
     constructor(collection: DocDBCollectionTreeItem, document: RetrievedDocument) {
         this._collection = collection;
         this._document = document;
-        this.partitionKeyValue = this.getPartitionKeyValue();
+        this._partitionKeyValue = this.getPartitionKeyValue();
+        this._label = getDocumentTreeItemLabel(this._document);
     }
 
     public get id(): string {
         return this.document.id;
     }
 
-    public get label(): string {
-        return this.document.id;
+    public async refreshLabel() {
```

Comment:
return value
no need to be async

Reply:
Good observation. 

IAzureTreeItem asks for a `refreshLabel()` that returns a `Promise<void>` Not having it async or a return type of `void` instead of `Promise<void>` gives type errors.

Reply:
I'll add the return type on the function header to make it clearer. :)